source/guides/stacks/dbrs.rst

```diff
@@ -0,0 +1,599 @@
+* BIOS with Reference Code
+* Intel Optane DC persistent memory
+
+Hardware configuration used in stacks development
+=================================================
+
+* Intel Server System R2208WFTZSR
+* BIOS with Reference Code
+  * BIOS ID: SE5C620.86B.0D.01.0438.032620191658
+  * BMC Firmware: 1.94.6b42b91d
+  * Apache Pass Firmware: 1.2.0.5310
+* 2x Intel Xeon Platinum 8268 Processor
+* Intel SSD DC S5600 Series 960GB 2.5in SATA Drive
+* 64 GB RAM - Distributed in 4x 16 GB DDR4 DIMM's
+* 2x Intel Optane DC Persistent Memory 256GB Module
+* 1-1-1 Layout 8 AEP : 1 RAM ratio
+
+
+.. list-table:: **Table 1. IMC**
+   :widths: 16,16,16,16,16,16
+   :header-rows: 1
+
+   * - Channel 2
+     - Channel 2
+     - Channel 1
+     - Channel 1
+     - Channel 0
+     - Channel 0
+
+    * - Slot 1
+      - Slot 0
+      - Slot 1
+      - Slot 0
+      - Slot 1
+      - Slot 0
+
+    * -
+      - 256 AEP
+      -
+      - 16 GB DRAM
+      -
+      - 16 GB DRAM
+
+
+
+
+
+Firmware configuration
+**********************
+
+.. important::
+
+   When updating DCPMM FW, all DCPMM samples must be in the same mode (you cannot mix 1LM and 2LM samples).
+
+The latest firmware download for the Intel Server System S2600WF Family is avaliable at the `Intel Download Center`_
+
+Firmware Update Steps
+=====================
+
+#. Unzip the contents of the update package and copy all files to the root directory of a removable media (USB flash drive).
+#. Insert the USB flash drive to any available USB port on the system to be updated.
+#. Boot to EFI shell.
+#. Input "fsx(x:0,1,...):" to enter into your usb disk
+#. Run "startup.nsh"
+#. After update BMC firmware, system BIOS, ME firmware,FD, FRUSDR, system will reboot automatically.
+
+
+If Intel Optane DC Persistent Memory is installed, run startup.nsh a second time after the first reboot to upgrade Intel Optane DC Persistent Memory Firmware:
+
+* Boot to EFI shell.
+* Input "fsx(x:0,1,...):" to enter into your usb disk
+* Run "startup.nsh" again to update the corresponding AEP FW.
+
+
+Hardware Configuration
+**********************
+
+
+Online Resources
+================
+
+Before going through the configuration steps, we strongly recommend visiting the following resources and wikis to have a broader understanding of what is being done
+
+* `Quick Start Guide`_ Configure Intel Optane DC Persistent Memory Modules on Linux
+* `Managing NVDIMMs`_
+* `Configure, Manage, and Profile`_ Intel Optane DC Persistent Memory Modules
+
+Optane DIMM Configuration
+=========================
+
+The persistent memory DIMMs can be configured in devdax or fsdax mode. The use case to enable database stack on a kubernetes environment currently only support fsdax mode.
+
+Configuration Steps
+===================
+
+.. important::
+
+Run the following steps with root privileges (sudo) as shown in the examples
```

Comment:
Left-align "Run.." with start of important above it.

Reply:
Aligned.  Don't know how I missed that.